packages/core/useFetch/index.ts

```diff
@@ -407,9 +412,10 @@ export function useFetch<T>(url: MaybeRef<string>, ...args: any[]): UseFetchRetu
         config.method = method
         config.payload = payload
         config.payloadType = payloadType
+        watchPayload()
         // Set the payload to json type only if it's not provided and a literal object is provided
         // The only case we can deduce the content type and `fetch` can't
-        if (!payloadType && payload && Object.getPrototypeOf(payload) === Object.prototype)
+        if (!isRef(payload) && !payloadType && payload && Object.getPrototypeOf(payload) === Object.prototype)
```

Comment:
What does this do?

Why not:

```suggestion
        if (!payloadType && unref(payload) && Object.getPrototypeOf(unref(payload)) === Object.prototype)
```

Reply:
In fact, 'Object.getPrototypeOf(payload) === Object.prototype' will always return false if the payload is a ref